client/trino-jdbc/src/main/java/io/trino/jdbc/AbstractConnectionProperty.java

```diff
@@ -146,21 +146,29 @@ public void validate(Properties properties)
 
     protected static final Predicate<Properties> ALLOWED = properties -> true;
 
-    interface Converter<T>
+    interface Converter<V, T>
     {
-        T convert(String value);
+        T convert(V value);
+
+        default boolean isEmpty(V value)
+        {
+            if (value instanceof String) {
+                return ((String) value).isEmpty();
+            }
+            return false;
```

Comment:
Should converter handle this logic ? It is more about Value rather than the converter.

Reply:
Smth like this https://github.com/trinodb/trino/pull/17853/commits/233fde95519075369aae726a20733ce165b72fea ?